Client/src/js/views/OwnersDetail.jsx

```diff
@@ -487,14 +492,11 @@ var OwnersDetail = React.createClass({
                 }
 
                 var headers = [
-                  { field: 'name',              title: 'Name'  },
-                  { field: 'phone',             title: 'Phone' },
-                  { field: 'mobilePhoneNumber', title: 'Cell'  },
```

Comment:
Don't remove `mobilePhoneNumber` it was [added](https://github.com/bcgov/hets/commit/31b131366b5f9fde7f491c524eb131030a9f67a9#diff-2802dc2988c7ac3ef836e6e0ca7183a6R490) as part of HETS-1020